README.md

```diff
@@ -46,6 +46,14 @@ bash download_dataset.sh {GCS_PROJECT} {GCS_BUCKET_NAME}
 
 To run maxtext the TPUVMs must have permission to read the gcs bucket. These permissions are granted by service account roles, such as the `STORAGE ADMIN` role. 
 
+## Getting Started: Configure Diagnostic Settings
+Here is the related PyPI package: https://pypi.org/project/cloud-tpu-diagnostics.
+### Debugging
+The following configurations will help to debug a fault or when a program is stuck or hung somewhere by collecting stack traces. Change the parameter values accordingly in `MaxText/configs/base.yml`:
+1. Set `collect_stack_trace: True` to enable collection of stack traces on faults or when the program is hung. To disable this, set `collect_stack_trace: False`.
+2. Set `stack_trace_to_cloud: False` to display stack traces on console. `stack_trace_to_cloud: True` will create a temporary file in `/tmp/debugging` in the TPUs to store the stack traces. There is an agent running on TPU VMs that will periodically upload the traces from the temporary directory to cloud logging in the gcp project.
```

Comment:
The /tmp/debugging logging is cool, I didn't realize this. For my own understanding, how is this logging achieved? Are you implementing an agent  in cloud-tpu-diagnostics to /tmp/debugging or is there a pre-existing agent that does this? Is it an ops agent?

Reply:
We have a module in `cloud-tpu-diagnostics` that collects stack traces and dump it to a temp file in TPU VMs. Then, there is a runtime monitor agent running on VMs that periodically uploads the traces from VMs to cloud logging.